WooCommerce/Classes/ViewModels/Order Details/OrderDetailsDataSource.swift

```diff
@@ -49,7 +49,6 @@ final class OrderDetailsDataSource: NSObject {
             isOrderCurrencyEligibleForCardPayment() &&
             isOrderStatusEligibleForCardPayment() &&
             isOrderPaymentMethodEligibleForCardPayment() &&
-            hasCardPresentEligiblePaymentGatewayAccount() &&
```

Comment:
As you mentioned in the PR description, it might be a good idea to revert this change before merging

Reply:
Sure. I'll just keep adding it back with each PR